diff --git a/target/linux/ar71xx/image/legacy-devices.mk b/target/linux/ar71xx/image/legacy-devices.mk
index 7723024..90855e9 100644
--- a/target/linux/ar71xx/image/legacy-devices.mk
+++ b/target/linux/ar71xx/image/legacy-devices.mk
@@ -1,7 +1,7 @@
 define LegacyDevice/A60
    DEVICE_TITLE := OpenMesh A40/A60
    DEVICE_PACKAGES := om-watchdog kmod-ath10k ath10k-firmware-qca988x \
-	mod-usb-core kmod-usb2
+	kmod-usb-core kmod-usb2
 endef
 LEGACY_DEVICES += A60
 
